client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/ClinicalDetails.tsx

```diff
@@ -99,6 +99,9 @@ const ClinicalDetails: React.FC<Props> = ({ id }: Props): JSX.Element => {
                     investigation: epidemiologyNumber,
                     user: userId
                 });
+                ClinicalDetailsSchema(validationDate).isValid(values).then(valid => {
```

Comment:
why move it to the then block, maybe you should create finally block?

Reply:
fixed